lmdeploy/archs.py

```diff
@@ -193,3 +193,21 @@ def get_model_arch(model_path: str):
             raise RuntimeError(
                 f'Could not find model architecture from config: {_cfg}')
         return arch, cfg
+
+
+def get_quantization_config(config):
+    """get quantization config from a model's config."""
+    if isinstance(config, dict):
+        for k, v in config.items():
+            if k == 'quantization_config':
+                return v
+            if isinstance(v, (dict, list)):
+                result = get_quantization_config(v)
+                if result is not None:
+                    return result
+    elif isinstance(config, list):
```

Comment:
Is it possible that `quantization_config` is inside a list? Seems `find_quantization_config(nested, target_key)` is a more general implementation.

Reply:
`get_quantization_config` is implanted from `find_quantization_config` except that `target_key` is hardcoded 'quantization_config`

Reply:
Yes, I mean the original function is a general function that we can also get other configurations through `target_key` such as `llm_config` or `language_config`.

Reply:
I see. In that case, I'd better change the function name, too